.github/workflows/docker.yml

```diff
@@ -108,8 +108,8 @@ jobs:
           context: .
           load: true
           tags: brew
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+          cache-from: type=registry,ref=ghcr.io/homebrew/ubuntu${{ matrix.version }}:cache
+          cache-to: type=registry,ref=ghcr.io/homebrew/ubuntu${{ matrix.version }}:cache,mode=max
```

Comment:
I assume this only works after the login step and should still use `gha` when running from a third-party PR branch.